UnitTesting-master/src/test/java/myTest.java

```diff
@@ -0,0 +1,155 @@
+import com.google.gson.Gson;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.EmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+import parser.JsonParser;
+import parser.NoSuchFileException;
+import parser.Parser;
+import shop.Cart;
+import shop.RealItem;
+import shop.VirtualItem;
+import java.io.ByteArrayOutputStream;
+import java.io.File;
+import java.io.IOException;
+import java.io.PrintStream;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+
+public class MyTest {
```

Comment:
Let's split this class into several classes (I think you can group them by @Tag value)

Reply:
I've separated MyTest class into 4 test classes.